client/my-sites/checkout/checkout-thank-you/index.jsx

```diff
@@ -428,7 +432,9 @@ export class CheckoutThankYou extends React.Component {
 			);
 		}
 
-		if ( wasEcommercePlanPurchased ) {
+		if ( wasMarketplaceProduct ) {
+			return <MarketplaceThankYou purchases={ purchases } />;
```

Comment:
Can we load this async? It has added an observable amount of bundle size:

```
name                      parsed_size           gzip_size
checkout                     +32960 B  (+2.5%)    +7909 B  (+2.3%)
```

Reply:
Fixed by 14bc6d9